jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/TaskVariablesQueryServiceTest.java

```diff
@@ -579,7 +578,95 @@ public void testTaskVariableQueryOnBigTaskSetAsJPA() throws Exception {
         
 
     }
-    
+
+
+    @Test
+    public void testUserTaskInstanceWithCustomVarsQueryMapperResult() throws Exception {
```

Comment:
This test is part of an "ignore" annotated class, so won't be executed in CI. Anyway, you can keep it for reference